src/firefly/js/visualize/ui/ImageSelectPanelResult.js

```diff
@@ -323,15 +323,21 @@ export function resultSuccess(plotInfo, hideDropdown = false) {
             });
         }
 
+
+        if (hideDropdown) {
+            dispatchHideDropDownUi();
+        }
+
         if (plotInfo.isThreeColor) {
             dispatchPlotImage(nPlotId, wpSet, true);
         } else {
             dispatchPlotImage(nPlotId, wpSet[0]);
         }
 
-
+/*
```

Comment:
nit picky- If you think you need to leave this code in you should probably use the // syntax.